exporter/topology_info_test.go

```diff
@@ -29,19 +29,75 @@ import (
 )
 
 func TestTopologyLabels(t *testing.T) {
-	ctx, cancel := context.WithTimeout(context.Background(), time.Second)
-	defer cancel()
+	tests := []struct {
+		containerName string
+		want          map[string]string
+	}{
+		{
+			containerName: "mongos",
+			want: map[string]string{
+				labelReplicasetName:  "",
+				labelReplicasetState: "0",
+				labelClusterRole:     "mongos",
+				labelClusterID:       "q",
+			},
+		},
+		{
+			containerName: "mongo-1-1",
+			want: map[string]string{
+				labelReplicasetName:  "rs1",
+				labelReplicasetState: "1",
+				labelClusterRole:     "shardsvr",
+				labelClusterID:       "d",
+			},
+		},
+		{
+			containerName: "mongo-cnf-1",
+			want: map[string]string{
+				labelReplicasetName:  "cnf-serv",
+				labelReplicasetState: "1",
+				labelClusterRole:     "configsvr",
+				labelClusterID:       "f",
+			},
+		},
+		{
+			containerName: "mongo-1-arbiter",
+			want: map[string]string{
+				labelReplicasetName:  "",
+				labelReplicasetState: "0",
```

Comment:
As per docs, this should be [7](https://docs.mongodb.com/v4.4/reference/replica-states/). Correct me if I'm wrong.

Reply:
No, because we are getting it from `getDIagnosticData` but `getDiagnosticData` doesn't work on arbiters so we are getting a 0